packages/core/src/Stores/Helpers/client.js

```diff
@@ -0,0 +1,11 @@
+// TODO: [move-to-shared] - Remove the implementation in ClientBase and add this to shared utils
+export const getClientAccountType = (loginid) => {
```

Comment:
Why not getting the account type based on landing company?

Reply:
I believe the logic was easier if account type was gotten from loginid and that has stayed.

Reply:
@negar-binary agree with this?

Reply:
it's the same that we have in binary-static, we can try changing it to landing company now, maybe at that time we didn't have a solution to use landing company in some cases